src/libs/actions/Report.js

```diff
@@ -315,6 +315,7 @@ function fetchChatReportsByIDs(chatList) {
             // than updating props for each report and re-rendering had merge been used.
             Onyx.mergeCollection(ONYXKEYS.COLLECTION.REPORT_IOUS, reportIOUData);
             Onyx.mergeCollection(ONYXKEYS.COLLECTION.REPORT, simplifiedReports);
+            Onyx.merge(ONYXKEYS.APP_DATA_LOADED, true);
```

Comment:
Any reason we're using `merge` and not `set` here? Is it to prevent a call to `keyChanged`?

Reply:
No reason in particular `merge()` and `set()` will have the same behavior when the value we are setting is simple like this.